components/shared_data_model/src/shared_data_model/metrics.py

```diff
@@ -113,6 +113,18 @@
             unit=Unit.ISSUES,
             sources=["azure_devops", "jira", "manual_number", "trello"],
         ),
+        job_runs_within_time_period=dict(
+            name="Job runs within time period",
+            description="The number of job runs within a specified time period.",
+            rationale="Frequent deployments are associated with smaller, less risky deployments and with lower lead "
+            "times for new features.",
+            unit=Unit.CI_JOB_RUNS,
+            direction=Direction.MORE_IS_BETTER,
+            target="30",
+            near_target="25",
```

Comment:
These numbers seem pretty high given a default lookback period of 90 days?

Reply:
For a daily build it would be pretty low, but maybe for a weekly build target 10 and near 8 ?